Event_schema.json

```diff
@@ -135,8 +135,13 @@
                 "tugsStandby.et.portAuthority",
                 "vtsArea.ata.vessel",
                 "vtsArea.atd.vessel",
+                "waste.atc.serviceProvider",
                 "waste.atc.vessel",
-                "waste.ats.vessel"
+                "waste.ats.serviceProvider",
+                "waste.ats.vessel",
+                "waste.cancel.serviceProvider",
+                "waste.etc.serviceProvider",
+                "waste.ets.serviceProvider"
```

Comment:
Apparently we forgot to update the scheme with `generate_json_schema.sh` the last time we changed the `Event_spec.ts`.